### PR TITLE
Basic routing, Navbar and Homepage

### PR DESCRIPTION
**Pull request by bhavant**
Added Routing
Navbar with Button links
Homepage view

review comments pls..

### CODE REVIEW

File: client/src/views/Homepage/Homepage.view.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function Homepage(props) {
+	return (
+		<div>
+			<h1>Welcome to Project Paycheck</h1>
+		</div>
+	)
+}
```
Comment (ghost) [inline client/src/views/Homepage/Homepage.view.jsx]: Can you make sure you have the Prettier extension installed. It'll keep indentation and code styling consistent across all contributors.

File: client/src/components/MainAppBar/MainAppBar.component.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import { Link as RouterLink } from 'react-router-dom';
+// import Link from '@material-ui/core/Link';
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+// import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+// import IconButton from "@material-ui/core/IconButton";
+// import MenuIcon from "@material-ui/icons/Menu";
+
```
Comment (ghost) [inline client/src/components/MainAppBar/MainAppBar.component.jsx]: In general it's frowned upon to include commented-out code. If something is not being used it's best to remove it.

File: client/src/App.js [hunk 3]
```diff
@@ -1,12 +1,24 @@
 import React from "react";
 import CssBaseline from "@material-ui/core/CssBaseline";
+
+import { Switch, Route } from 'react-router-dom';
+
+// View Imports
+import Homepage from "./views/Homepage/Homepage.view"
```
Comment (ghost) [inline client/src/App.js]: `Homepage.view` not sure why you're including these sub names/identifiers. If you can make a good argument why we should do that, I would be okay with it, but it's just not a common React way of doing things. I'd like to be consistent throughout the codebase.

**Review by ghost — CHANGES_REQUESTED** [review]
Nice PR, I just left some comments. If you have any questions feel free to message me

File: client/src/App.js (hunk 3 above)
Comment (bhavant) [reply client/src/App.js]: Force of habit from work! We use JS not React but have a naming convention to follow. I will change this.

File: client/src/components/MainAppBar/MainAppBar.component.jsx (hunk 2 above)
Comment (bhavant) [reply client/src/components/MainAppBar/MainAppBar.component.jsx]: Removed the commented code.

File: client/src/views/Homepage/Homepage.view.jsx (hunk 1 above)
Comment (bhavant) [reply client/src/views/Homepage/Homepage.view.jsx]: Configured Prettier. It chooses not to work on new files! Works when close and open. I will check further. This file was corrected.

**Pull request merged**